model/src/main/kotlin/config/LicenseFindingCuration.kt

```diff
@@ -27,10 +27,10 @@ import com.fasterxml.jackson.databind.util.StdConverter
 import org.ossreviewtoolkit.spdx.SpdxExpression
 
 /**
- * A curation for license findings.
+ * A curation for a license finding. Use this to correct an existing license finding, to mark an existing finding as
```

Comment:
I propose to simplify this sentence to just "A curation for a license finding. Use it to correct a license finding or to add a license that was not previously detected."

Reply:
Fine with me.